kolena/annotation.py

```diff
@@ -91,8 +91,8 @@ def _data_type() -> _AnnotationType:
         return _AnnotationType.BOUNDING_BOX
 
     def __post_init__(self) -> None:
-        object.__setattr__(self, "width", self.bottom_right[0] - self.top_left[0])
```

Comment:
what features on our platform does this break? If not, I think it's better to show the data error then hide them.

Reply:
If it's an actual data error, it should still be noticeable in the UI -- this just makes us robust to a easy to make mistake for bbox input where the resolution should be straightforward. We've had users run into this, and I'd prefer to avoid explaining that they need to re-push their data to fix something like this.